src/components/Navbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Navbar.css';
+
+const Navbar = () => {
+  const linkStyle = {
+    margin: '1rem',
+    textDecoration: 'none',
+    color: 'rgb(17, 168, 161)',
+  };
+  return (
+    <>
+      <ul className="header">
+        <li>
+          <NavLink style={linkStyle} to="/">
+            Home
+          </NavLink>
+        </li>
+        <li>
+          <NavLink style={linkStyle} to="/calculator">
+            Calculator
+          </NavLink>
+        </li>
+        <li>
+          <NavLink style={linkStyle} to="/quote">
+            Quote
+          </NavLink>
+        </li>
+      </ul>
```

Comment:
[OPTIONAL] Your UI can be improved if you aligned the menu items with the logo so that all the navbar items are perfectly displayed at the top of the page.

Reply:
Ok, I will. Thanks so much.